### PR TITLE
[FEATURE] Create verifications table

### PR DESCRIPTION
**Pull request by wilsonianb**
DEC-470

### CODE REVIEW

File: api/models/verification.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+'use strict';
+module.exports = function(sequelize, DataTypes) {
+  var Verifications = sequelize.define('Verifications', {
+    validation_public_key: {
+      type: DataTypes.STRING,
+      allowNull: false
```
Comment (stevenzeiler) [inline api/models/verification.js]: Should we validate the length of validation_public_key?

Comment (wilsonianb) [inline api/models/verification.js]: Sure

Comment (wilsonianb) [inline api/models/verification.js]: Added validation for `validation_public_key` in Validations and Verifications tables.

**Comment by stevenzeiler** [comment]
:+1:

**Comment by wilsonianb** [comment]
:+1:

**Pull request merged**